ygot/pathstrings.go

```diff
@@ -43,7 +43,7 @@ const (
 // representing the path.
```

Comment:
Suggest that we add a function comment here that says that the supplied `path` is always parsed as absolute and therefore should be the concatenation of the `prefix` plus `path` if a `prefix` exists.

Reply:
Clarified that path is absolute.
`prefix` is a property of Notification, and is not something this package deals with directly.